packages/lesswrong/components/common/withHover.jsx

```diff
@@ -4,7 +4,7 @@ const withHover = (WrappedComponent) => {
   return class HoverableComponent extends Component {
     state = { hover: false, anchorEl: null }
 
-    handleMouseEnter = (event) => {
+    handleMouseMove = (event) => {
       this.setState({ hover: true, anchorEl: event.currentTarget})
```

Comment:
Is `anchorEl` used by any of this element's children? It's doing something very counterintuitive; it's the deepest-in-tree element that the mouse is over, which is usually not what you want.

Reply:
That seems true, my guess is we actually want to have a reference to the div that is introduced by the `withHover` component, which we can be sure doesn't change.